src/main.js

```diff
@@ -430,10 +430,12 @@ function handleAppWebContentsCreated(dc, contents) {
     event.preventDefault();
     if (!isTrustedURL(url)) {
       log.info(`Untrusted popup window blocked: ${url}`);
+      shell.openExternal(url);
```

Comment:
Can we be sure that `url` has a trusted scheme here? I.e `http`, `https`, `mattermost`, or something from `allowedProtocols.json`.

Reply:
You mean for opening in the user browser? should we be restrictive about that?

Reply:
Yes. The problem is `shell.openExternal` doesn't _just_ open the default web browser, it's a potential execution sink. See e.g. https://github.com/doyensec/electronegativity/wiki/OPEN_EXTERNAL_JS_CHECK

Reply:
i'm removing this line, as apart from what you mention it also means that links open twice in the browser